services/feed-manager-service/feed-manager-controller/src/main/java/com/thinkbiganalytics/feedmgr/service/feed/FeedManagerFeedService.java

```diff
@@ -97,6 +100,20 @@
     Collection<? extends UIFeed> getFeeds(boolean verbose);
 
     /**
+     * Return a list of feeds, optionally returning a more verbose object populating all the templates and properties.
+     * Verbose will return {@link FeedMetadata} objects, false will return {@link FeedSummary} objects
+     * <p>
+     * The sized of the returned list will not be greater than the limit parameter, and the first element 
+     * of the list will be the n'th feed in the list of all feeds as specified by the start parameter.
+     *
+     * @param verbose true will return {@link FeedMetadata} objects, false will return {@link FeedSummary} objects
+     * @param pageable describes the page requested
+     * @param filter TODO
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)